### PR TITLE
feat: screen recorder

### PR DESCRIPTION
**Pull request by heyumoAi**
Description: interface implementation

Log: dtk scree recorder interface implementation

### CODE REVIEW

File: src/multimedia/screenrecorder/videostreaminterface.cpp [hunk 1]
```diff
@@ -3,21 +3,39 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
```
Comment (xzl01) [inline src/multimedia/screenrecorder/videostreaminterface.cpp:3]: 这个不对

**Review by Groveer — COMMENTED** [review]
+1

File: src/multimedia/screenrecorder/videostreaminterface.cpp (hunk 1 above)
Comment (xzl01) [reply src/multimedia/screenrecorder/videostreaminterface.cpp:3]: 注意下spdx头的格式

**Review by xzl01 — COMMENTED** [review]
+1

**Review by toberyan — APPROVED** [review]
+1

**Pull request merged**